### ISSUE TITLE
Make it easier to distinguish user with the same name in the share drop down

### ISSUE THREAD

**Issue by schiessle** [nextcloud/server#15041]
If you have many users called "John Doe" and you type "John" in the share dialog you see a lot of users with the same user name. On hover we show the username but this is also often not really useful. Especially if the UID is a cryptic LDAP UUID for example. I think it would make more sense to show more user friendly information, for example the email address would be a useful information to distinguish the users in many situations.

@nextcloud/designers

**Comment by jancborchardt** [comment]
Yes, if there’s an email address available, we should show that in the second row in the suggestion dropdown. (Just like we do the second row for "Add to project".

Probably needs @nextcloud/javascript?

**Comment by blizzz** [comment]
@schiessle https://docs.nextcloud.com/server/15/admin_manual/configuration_user/user_auth_ldap.html#directory-settings _2nd User Display Name Field_

**Comment by blizzz** [comment]
> On hover we show the username 

this we should not do. It is not an end user facing value.

**Comment by schiessle** [comment]
I had another user asking for it. Is there any chance to get this implemented in a generic way, not only for LDAP where we have the "hack" with the 2nd Display Name?

**Comment by skjnldsv** [comment]
Yep, we agree here, just not a priority (yet), this is set to develop :)

**Comment by Dubidubiduu** [comment]
Would also appreciate to get some further information about the user, based on his privacy settings, e.x. Address (could be ZIP, Town) but also his groups and maybe a department (which would be an additional field).

**Comment by PVince81** [comment]
see https://github.com/owncloud/core/issues/28979 and https://github.com/owncloud/core/pull/29334

bring in a new field that is returned by share autocomplete, something to display in brackets

and have an admin setting to configure what value to display there
